pallets/zk-snarks/sample/vk.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
What's the purpose of this file? It's not valid verification key. Same thing relates to proof file added in this pull request.

Reply:
You are right, I will remove commit "Improving documentation" from this PR. I will need to create a separate PR and prepare a documentation for this.